client/state/README.md

```diff
@@ -205,3 +216,9 @@ ageReducer( -5, { type: DESERIALIZE } ) === -5
 age( -5, { type: DESERIALIZE } ) === 0
 age( 23, { type: DESERIALIZE } ) === 23
 ```
+
+### combineReducers( reducersObject )
+This has a compatible function signature with redux's famous combineReducers function.
```

Comment:
"with redux's famous combineReducers function" seems a bit awkward here. what if we tried to be less flowery and laid it bare with something like this…

> A wrapper around Redux's `combineReducers()` which shares an identical function signature.